test/sut/tests/homepage.bats

```diff
@@ -21,6 +21,36 @@ setup () {
   #webrouter_header_size="1024"
 }
 
+@test "homepage: http homepage request with query goes to WordPress" {
+  test_web http "$BUWEBHOST" '/?page_id=2002'
+  assert_status 200
+  # eventually it should be 
+  #assert_backend "content"
+  # right now it can be one of two values - legacy for Solaris servers and content for all others
+  # 6/26/2018: now that we have transitioned it can only be 
```

Comment:
Maybe streamline the comments here to only include the up-to-date use case?  Also I think we removed `aws_home_index` and only `aws_home` is still relevant?